### ISSUE TITLE
Unable to set header value to 0

### ISSUE THREAD

**Issue by jgrocott** [phalcon/cphalcon#11026]
Hi,

I'm developing a REST API using the Micro framework. I have an $app->before() event which performs some throttling. If the configured hit rate is exceeded by a user I'm sending a HTTP status code of 429 (Too Many Requests). I'm also attempting to set a 'X-RateLimit-Remaining' HTTP header which contains the number of remaining API calls for the current user. This works successfully until the value of a header reaches 0. When this value reaches 0, a HTTP status of 500 is returned instead of 429. Manually setting the header value using header() works OK.

OS: Ubuntu 14.04.1 LTS
Phalcon: 2.0
PHP: 5.6.14

Code to reproduce the bug;

```php 
<?php

$app->before(function() use ($app)
{
    $app->response->setHeader("X-RateLimit-Remaining", 0); // This fails
    //header("X-RateLimit-Remaining:  0"); // This works
    $app->response->setStatusCode(429, "Too Many Requests")->sendHeaders();
    return false;
});

$app->get('/', function () use ($app) {
    echo $app['view']->render('index');
});

$app->notFound(function () use ($app) {
    $app->response->setStatusCode(404, "Not Found")->sendHeaders();
    echo $app['view']->render('404');
});

**Comment by andresgutierrez** [comment]
Can you try again passing a string?

```php
$app->response->setHeader("X-RateLimit-Remaining", "0");
```

**Comment by jgrocott** [comment]
Hi Andy,

```php
$app->response->setHeader("X-RateLimit-Remaining", "0"); // Fails
$app->response->setHeader("X-RateLimit-Remaining", "0 "); // Added a space to the end and it's successful
```

**Comment by andresgutierrez** [comment]
Fixed in 2.1.x branch
